cmd/cloudFoundryCreateService.go

```diff
@@ -0,0 +1,103 @@
+package cmd
+
+import (
+	"fmt"
+
+	"github.com/SAP/jenkins-library/pkg/cloudfoundry"
+	"github.com/SAP/jenkins-library/pkg/command"
+	"github.com/SAP/jenkins-library/pkg/log"
+	"github.com/SAP/jenkins-library/pkg/telemetry"
+	"github.com/pkg/errors"
+)
+
+func cloudFoundryCreateService(config cloudFoundryCreateServiceOptions, telemetryData *telemetry.CustomData) {
+
+	cf := cloudfoundry.CFUtils{Exec: &command.Command{}}
```

Comment:
Ok, I have refactored related code and moved the business logic to `runCloudFoundryCreateService` function. Could you re-check if this is appropriate now?

Reply:
Ok, I have refactored related code and moved the business logic to `runCloudFoundryCreateService` function. Could you re-check if this is appropriate now?

Reply:
I personally would handle that even stronger. The signature you get when the code is generated is

```
runCloudFoundryCreateService(config *cloudFoundryCreateServiceOptions, telemetryData *telemetry.CustomData, command execRunner)
```

This signature has been adjusted manually. That is possible, of course. But I personally would change that so that the signature does not change. I think it is easier to understand when the signature is not changed (... also compared to other steps). It's up to you if you would like to adjust again ...

The `cloudfoundry.CFUtils` can also be instanciated in the the other method which gets called. It looks strange to set the command line handler on `cloudfoundry.CFUtils` and to extract it from that again later (var c = cf.Exec). Looks rough. Why not handing over the command line handler into the method and instancitate the more specific cf util inisde the method which gets called from the generated code?